louis/db/api/__init__.py

```diff
@@ -1,5 +1,31 @@
 from louis.models import openai
+import os
+import json
+import sys
+import louis.db as db
+import dotenv
+# This is used to load the .env file
+dotenv.load_dotenv()
 
+FINESSE_WEIGHTS = os.environ.get("FINESSE_WEIGHTS")
+
+# should be something like that : '{
+#     "recency": 1,
+#     "traffic": 1,
+#     "current": 0.5,
+#     "typicality": 0.2,
+#     "similarity": 1
+# }'
+
+if FINESSE_WEIGHTS:
+    if FINESSE_WEIGHTS == 'finesse-weights.json':
```

Comment:
use os.path.exists() instead of expecting a specific filename

Reply:
you're right, it's better this way, thanks